### ISSUE TITLE
Upgrade to version 1.16.0 generates error CS0426

### ISSUE THREAD

**Issue by hulken76** [Azure/azure-functions-dotnet-worker#2028]
After upgrading to version 1.16.0 I get build error on all my azure functions:

CS0426 - The type name 'ExampleFunction' does not exist in the type 'ExampleFunction' - GeneratedFunctionExecutor.g.cs	

VS:2022
SDK:7.0.403

Works as expected if i downgrade to version 1.15.1

**Comment by kshyju** [comment]
@hulken76 Do you happen to have the same value("ExampleFunction") for the namespace and your class name? If so, you are experiencing this known issue #1990 . We have a [fix coming out soon](https://github.com/Azure/azure-functions-dotnet-worker/pull/1993). Until then, please add the below entry to your csproj file to explicitly specify a namespace to be used for the auto generated types.

```xml
<FunctionsGeneratedCodeNamespace >YourProjectNamespace</FunctionsGeneratedCodeNamespace>
```

**Comment by fabiocav** [comment]
Closing this as it should be resolved by #1990
